gson/src/main/java/com/google/gson/internal/bind/ReflectiveTypeAdapterFactory.java

```diff
@@ -63,13 +63,13 @@ static boolean excludeField(Field f, boolean serialize, Excluder excluder) {
     return !excluder.excludeClass(f.getType(), serialize) && !excluder.excludeField(f, serialize);
   }
```

Comment:
I am wondering may be should do a release that breaks binary compatibility with this and a few additional features (such as implement List for JsonArray).

What's the big downside of breaking binary compatibility.